packages/app/features/settings/AppInfoScreen.tsx

```diff
@@ -1,38 +1,110 @@
 import { NativeStackScreenProps } from '@react-navigation/native-stack';
+import { useDebugStore } from '@tloncorp/shared';
+import { getCurrentUserId } from '@tloncorp/shared/dist/api';
 import * as store from '@tloncorp/shared/dist/store';
 import {
   AppSetting,
+  Button,
   ListItem,
   ScreenHeader,
   SizableText,
   Stack,
+  Text,
   View,
+  XStack,
   YStack,
 } from '@tloncorp/ui';
 import { preSig } from '@urbit/aura';
 import * as Application from 'expo-application';
 import * as Updates from 'expo-updates';
-import { useMemo } from 'react';
+import { useEffect, useMemo, useState } from 'react';
 import { useCallback } from 'react';
-import { Platform } from 'react-native';
+import { Alert, Platform, Switch } from 'react-native';
+import { getEmailClients, openComposer } from 'react-native-email-link';
 import { ScrollView } from 'react-native-gesture-handler';
 
 import { NOTIFY_PROVIDER, NOTIFY_SERVICE } from '../../constants';
+import { toggleDebug } from '../../lib/debug';
 import { getEasUpdateDisplay } from '../../lib/platformHelpers';
 import { RootStackParamList } from '../../navigation/types';
 
 const BUILD_VERSION = `${Platform.OS === 'ios' ? 'iOS' : 'Android'} ${Application.nativeBuildVersion}`;
 
 type Props = NativeStackScreenProps<RootStackParamList, 'AppInfo'>;
 
+function makeDebugEmail(appInfo: any, platformInfo: any) {
+  return `
+----------------------------------------------
+Insert description of problem here.
+----------------------------------------------
+
+Tlon ID: ${getCurrentUserId()}
+
+Platform Information:
+${JSON.stringify(platformInfo)}
+
+App Information:
+${JSON.stringify(appInfo)}
+`;
+}
+
 export function AppInfoScreen(props: Props) {
   const { data: appInfo } = store.useAppInfo();
+  const { enabled, logs, logId, uploadLogs } = useDebugStore();
   const easUpdateDisplay = useMemo(() => getEasUpdateDisplay(Updates), []);
+  const [hasClients, setHasClients] = useState(true);
+
+  useEffect(() => {
+    async function checkClients() {
```

Comment:
nit, but might make sense to pull the email handling out into a hook rather than inlined on the screen

Reply:
if we start using this more yeah we should